manual/deploy/auto_login_seadrive.md

```diff
@@ -0,0 +1,70 @@
+# Auto Login to SeaDrive on Windows
+
+[Kerberos](https://web.mit.edu/kerberos/) is a widely used single sign on (SSO) protocol. Supporting of auto login will use a Kerberos service. For server configuration, please read the [Remote User](./remote_user.md).
+
+## Technical Details
+
+The client machine has to join the AD domain which belongs to the KDC node. Since the client machine has been authenticated by KDC when a Windows user logs in, a Kerberos ticket will be generated for current user without needs of another login in the brower.
+
+When a program using the WinHttp API tries to connect a server, it can perform a login automatically through the [Integrated Windows Authentication](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication). Internet Explorer follows this mechanism, as well as SeaDrive.
+
+The details of Integrated Windows Authentication is described below:
+
+1. Decide whether or not to use IWA according to the address and Internet Options. (more in next section)
+2. Send a request to the server (e.g. http://test.seafile.com/sso)
+3. The server returns an HTTP 401 unauthorized response with the Negotiate header which includes an authentication protocol.
+4. The WinHttp API will try to use Kerberos first, if there is a valid ticket from KDC. The request will be sent again, together with the ticket in an HTTP header.
+5. Then, Apache can check the ticket with KDC, and extract the username from it. The username will be passed to SeaHub for a successful auto login.
+6. If the WinHttp API failed to get a ticket, it will then try the NTLM protocol by sending an HTTP request with Negotiate NTLMSSP token in the header. Without supporting the NTLM protocol, Apache shall returns an HTTP 401 unauthorized response and stops negotiation. At this point, the brower will pop up a login dialog, which means an auto login failure.
+
+In short:
+
+1. The client machine has to join the AD domain.
+2. The Internet Options has to be configured properly.
+3. The WinHttp API should be able to get a valid ticket from KDC. (The address should be registered in KDC.)
```

Comment:
The WinHttp API should be able to get a valid ticket from KDC. Make sure you use the correct server address (e.g. test.seafile.com) when you generate keytab file on the domain controller.